content/library/api/widgets/page_link.md

```diff
@@ -0,0 +1,6 @@
+---
+title: st.page_link
+slug: /library/api-reference/widgets/st.page_link
+---
+
+<Autofunction function="streamlit.page_link" />
```

Comment:
Note for later follow-up: once we have the full nav guide, we might want to link it from here too?

Reply:
Added.